apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql

```diff
@@ -69,10 +74,10 @@ type AssociatedActors {
 
 type BalanceResponsibilityAgreement {
   meteringPointType: String
+  energysupplierName: ActorNameWithId
+  balanceresponsibleId: ActorNameWithId
```

Comment:
suggestion: can we call them `energySupplierWithName` and `balanceResponsibleWithName` instead? There's already something similar under `BalanceResponsibleType`